modules/libcom/src/iocsh/iocsh.cpp

```diff
@@ -1325,8 +1327,7 @@ iocshCmd (const char *cmd)
 int epicsStdCall
 iocshLoad(const char *pathname, const char *macros)
 {
-    if (pathname)
-        epicsEnvSet("IOCSH_STARTUP_SCRIPT", pathname);
+// IOCSH_STARTUP_SCRIPT is set inside iocshBody()
```

Comment:
I don't think this comment is needed. Documenting history of the code in code comments is important when the history affects the present. But this is not the case here, I doubt knowing that `IOCSH_STARTUP_SCRIPT` used to be set here once upon a time will help the future reader.